packages/wonder-blocks-dropdown/components/action-menu-opener-core.js

```diff
@@ -9,17 +9,31 @@ import {addStyle, View} from "@khanacademy/wonder-blocks-core";
 import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {Strut} from "@khanacademy/wonder-blocks-layout";
-
-import type {
-    ClickableHandlers,
-    ClickableState,
-} from "@khanacademy/wonder-blocks-core";
-import type {SharedProps} from "./action-menu-opener.js";
+import type {AriaProps, ClickableState} from "@khanacademy/wonder-blocks-core";
 
 type Props = {|
-    ...SharedProps,
```

Comment:
Is `SharedProps` still being used somewhere?

Reply:
I don’t think so